setup.py

```diff
@@ -36,6 +37,7 @@ def read_file(filename, encoding='utf8'):
     'pyasn1',  # urllib3 InsecurePlatformWarning (#304)
     'pycrypto',
     'PyOpenSSL',
+    'pyparsing>=1.5.5',
```

Comment:
It was copied from the pip requirements for nginxparser; see https://github.com/fatiherikli/nginxparser/pull/8 for a maybe-justification. From the release notes, it looks like 1.5.5 is the last "stable" release that is compatible with both python 2 and 3.

I was actually a little surprised that Let's Encrypt doesn't pin all versions in order to make the builds more predictable. Is there a reason for avoiding pins here?